nav2_collision_monitor/src/polygon.cpp

```diff
@@ -314,7 +314,7 @@ bool Polygon::getParameters(std::string & polygon_pub_topic, std::string & footp
     std::vector<double> poly_row =
       node->get_parameter(polygon_name_ + ".points").as_double_array();
     // Check for points format correctness
-    if (poly_row.size() <= 4 || poly_row.size() % 2 != 0) {
+    if (poly_row.size() <= 8 || poly_row.size() % 2 != 0) {
```

Comment:
6? For a triangle

Reply:
For a triangle it will be 8, since first and last vertex of this list should be the same

Reply:
> last vertex of this list should be the same

Oops. It looks like I was wrong somewhere in the beginning when understanding of polygons in ROS. The last point _should not_ be the same as first. Will fix it shortly.

Reply:
Fixed in https://github.com/ros-planning/navigation2/pull/2982/commits/d8f449cb35a93c36d69801c97949c093a7aabedb